src/server/src/Server.cpp

```diff
@@ -253,37 +291,41 @@ Server::State Server::get_state()
     return state;
 }
 
-
 void Server::stop_motion()
 {
     drdStop();
 }
 
-
```

Comment:
The adopted convention is having two spaces between each function, can you please put it back? :)

Reply:
done